lumicks/pylake/fitting/parameters.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from collections import OrderedDict
+from tabulate import tabulate
+
+
+class Parameter:
+    def __init__(self, value=0.0, lb=-np.inf, ub=np.inf, vary=True, init=None, shared=False, unit=None):
+        """Model parameter
+
+        Parameters
+        ----------
+        value: float
+        lb, ub: float
+        vary: boolean
+        init: float
+        shared: boolean
+        """
+        self.value = value
+        self.lb = lb
+        self.ub = ub
+        self.vary = vary
+        self.shared = shared
+        self.unit = unit
```

Comment:
I feel like this part is quite under-documented. And I feel like, if this part of the API is not going to be used very often, then it would be better to have more expressive names, e.g. `lb` -> `lower_bound`. The latter would be much clearer for someone who just happened to see this in someone else's code. In general, I think that name length should be inversely proportional to how often the thing is used.

Reply:
True. I've added a little extra docstring for parameters, and changed lb/ub to lower_bound and upper_bound.